PSReadLine/Prediction.Entry.cs

```diff
@@ -0,0 +1,351 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
The PSReadLine is licensed under `BSD 2-Clause "Simplified" License` instead of MIT. Shall I change the header to the following?
```
// Copyright (c) Microsoft Corporation.
// Licensed under the BSD 2-Clause "Simplified" License.
```

Reply:
Yes, let's use that

Reply:
Will address in a separate PR.